content/docs/start/experiments-trail/experiments-1.md

```diff
@@ -0,0 +1,334 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+<abbr>Experiments</abbr> proliferate quickly in ML projects where there are many
+parameters to tune or other permutations of the code. We can organize such
+projects and keep only what we ultimately need with `dvc experiments`. DVC can
+track experiments for you so there's no need to commit each one to Git. This way
+your repo doesn't become polluted with all of them. You can discard experiments
+once they're no longer needed.
+
+Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
+[compare the changes](/doc/start/metrics-parameters-plots). In this section, we
+will explore the basic features of DVC experiment management with
+[`get-started-experiments`][gse] project.
```

Comment:
Maybe it's possible to have a single intro paragraph? I'd try to summarize a little (e.g. "merge" sentences 1 and 2). Some specific notes:

- `permutations of the code`: complex terms for an intro.
- `You can discard experiments`: may be implicit in the concept of "experiment"
- `Previously, we learned`: Does it matter? I.e. are these new GS pages still mostly connected in a sequence?